ipaserver/install/server/install.py

```diff
@@ -1198,6 +1198,7 @@ def uninstall(installer):
     tasks.restore_hostname(fstore, sstore)
     tasks.restore_pkcs11_modules(fstore)
     fstore.restore_all_files()
+    tasks.reenable_resolved(sstore)
```

Comment:
restore_all_files() should be called last IIRC

Reply:
Fixed, ty.